main/src/library/Nel.flix

```diff
@@ -408,7 +408,7 @@ mod Nel {
             case _        => unreachable!()
         };
         let Nel(x, xs) = l;
-        loop(x :: xs, x -> x)
+        loop(x :: xs, id -> id)
```

Comment:
We have an identity function :) and even an eidentity :)

Reply:
Oh. You removed OK 👍 

But lets please not call an argument id. We can call it z or w.